sdk/communication/azure-communication-identity/tests/test_communication_identity_client.py

```diff
@@ -298,132 +315,143 @@ def test_get_token_with_no_scopes(self, communication_livetest_dynamic_connectio
         user = identity_client.create_user()
 
         with pytest.raises(Exception) as ex:
-            token_response = identity_client.get_token(user, scopes=None)
+            identity_client.get_token(user, scopes=None)
+
+        assert ex is not None
+        assert str(ex.value.error.code) == 'ValidationError'
 
-    
     @CommunicationPreparer()
+    @acs_identity_decorator
+    @recorded_by_proxy
     def test_get_token_for_teams_user_from_managed_identity(self, communication_livetest_dynamic_connection_string):
-        if(self.skip_get_token_for_teams_user_test()):
+        if skip_get_token_for_teams_user_test(self):
             return
         endpoint, access_key = parse_connection_str(communication_livetest_dynamic_connection_string)
         if not is_live():
             credential = FakeTokenCredential()
         else:
             credential = DefaultAzureCredential()
         identity_client = CommunicationIdentityClient(
-            endpoint, 
-            credential, 
+            endpoint,
+            credential,
             http_logging_policy=get_http_logging_policy()
         )
-        aad_token, user_object_id = self.generate_teams_user_aad_token() 
+        aad_token, user_object_id = generate_teams_user_aad_token(self)
         token_response = identity_client.get_token_for_teams_user(aad_token, self.m365_client_id, user_object_id)
         assert token_response.token is not None
 
     @CommunicationPreparer()
+    @acs_identity_decorator
+    @recorded_by_proxy
     def test_get_token_for_teams_user_with_valid_params(self, communication_livetest_dynamic_connection_string):
-        if(self.skip_get_token_for_teams_user_test()):
+        if skip_get_token_for_teams_user_test(self):
             return
         identity_client = CommunicationIdentityClient.from_connection_string(
             communication_livetest_dynamic_connection_string,
             http_logging_policy=get_http_logging_policy()
         )
-        aad_token, user_object_id = self.generate_teams_user_aad_token() 
+        aad_token, user_object_id = generate_teams_user_aad_token(self)
         token_response = identity_client.get_token_for_teams_user(aad_token, self.m365_client_id, user_object_id)
         assert token_response.token is not None
 
-    @parameterized.expand(
-        [
-            ("empty_token", ""),
-            ("invalid_token", "invalid")
-        ]
-    )
-    def test_get_token_for_teams_user_with_invalid_token(self, _, invalid_token):
-        if(self.skip_get_token_for_teams_user_test()):
+    @CommunicationPreparer()
+    @pytest.mark.parametrize("_, value", [("empty_token", ""), ("invalid_token", "invalid")])
+    @ArgumentPasser()
+    @acs_identity_decorator
+    @recorded_by_proxy
+    def test_get_token_for_teams_user_with_invalid_token(self, _, value):
```

Comment:
I suggest reverting back to the old name of the param, same comment for everywhere where it's changed to `value`

Reply:
Done